src/app/modules/account/components/update-request/update-request.component.html

```diff
@@ -54,6 +54,20 @@ <h2>Documents to support status in Canada</h2>
   <msp-support-documents
     [supportDocList]="nameChangeDueToMarriageDocs"
     [(supportDoc)]="person.updateNameDueToMarriageDocType">
+    <div class='immediate-tip-slot'>
+      <aside>
+        <div class="card">
+          <div class="card-body">
+            <div>Tip</div>
+            <div>
+              For name changes due to marriage, you may combine your birth last name with a
+              current and/or previous married last name(s). Last names may be combined in any
+              order with a hyphen (-) or a space..
```

Comment:
Minor comment - Is it supposed to have two period at the end of the last line.

Reply:
Good catch. It has been fixed.